client/commands/v2/pysa_server.py

```diff
@@ -52,13 +58,68 @@ def __init__(
         pyre_arguments: start.Arguments,
         binary_location: str,
         server_identifier: str,
+        file_tracker: dict,
```

Comment:
Given that you don't actually ever pass anything but an empty dict into this constructor, there isn't any value in accepting `file_tracker` as a parameter. Instead, you can just create an empty set (see my comment below) and assign it to `self.file_tracker` all within the constructor.
```suggestion
```